README.md

```diff
@@ -82,7 +94,8 @@ Describe the differences between unit and functional testing. What type of testi
 Your answer:
 ```text
 
-Replace this with your answer
+Functional testing test the bigger picture/end user interactions with the app. These test click on links and go to webpages.
+Unit testing, like RSpec, do small test. They test small chunks of code like a method in a class.  
 ```
```

Comment:
What you're referring to in your first answer is "acceptance testing." Functional testing checks to see if an application exhibits a specific functionality, which might require the testing of multiple methods.